modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -153,63 +151,59 @@ private static String prepareLogsFile() {
         return null;
     }
 
-    private String generateConfig(final String dataDir, final int networkId, final String keystoreDir, final String fleet, final Object upstreamConfig) throws JSONException {
+    private String generateConfig(final JSONObject defaultConfig, final String root, final String keystoreDir, final String fleet) throws JSONException {
+        // retrieve parameters from app config, that will be applied onto the Go-side config later on
+        final String dataDir = root + defaultConfig.get("DataDir");
```

Comment:
If we create a path, [`Paths.get`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get(java.lang.String,%20java.lang.String...)) could be used. I guess it will be `final String dataDir = Paths.get(root, defaultConfig.get("DataDir"))`.

Reply:
@PombeirP have you seen this ?